solidity/contracts/utility/ContractRegistryClient.sol

```diff
@@ -0,0 +1,110 @@
+pragma solidity 0.4.26;
+import './Owned.sol';
+import './Utils.sol';
+import './interfaces/IContractRegistry.sol';
+
+/**
+  * @dev Base contract for ContractRegistry clients
+*/
+contract ContractRegistryClient is Owned, Utils {
+    IContractRegistry public registry;      // address of the current contract-registry
+    IContractRegistry public prevRegistry;  // address of the previous contract-registry
+    bool public allowRegistryUpdate = true; // allow/prevent changing the contract-registry

```

Comment:
the comment isn't accurate - it restricts the change to the owner

Reply:
It allows/prevents the change to/from non-authorized callers.
In the case of `BancorX`, it's anyone who is not an owner.
In the case of `BancorConverter`, it's anyone who is neither an owner nor a manager.
This is determined by the implementation of the (internal) function `callerIsAuthorized`.

Reply:
Yup, though from a users (docs) perspective, "disabling" means that nobody can do that, not just a list of authorized users

Reply:
Can you please check the new terminology that I've applied in this file?